include/zephyr/sys/byteorder.h

```diff
@@ -17,21 +17,21 @@
 #include <zephyr/toolchain.h>
 
 /* Internal helpers only used by the sys_* APIs further below */
```

Comment:
I forgot to mention, we should probably remove this line, as we are not anymore keeping this as internal